Modules/_testcapimodule.c

```diff
@@ -3301,6 +3301,20 @@ getbuffer_with_null_view(PyObject* self, PyObject *obj)
     Py_RETURN_NONE;
 }
 
+/* PyBuffer_SizeFromFormat() */
+static PyObject *
+pybuffer_size_from_format(PyObject *self, PyObject *args)
+{
+    const char *format;
+
+    if (!PyArg_ParseTuple(args, "s:pybuffer_size_from_format",
+                          &format)) {
+        return NULL;
+    }
+
+    return PyLong_FromSsize_t(PyBuffer_SizeFromFormat(format));
```

Comment:
I don't believe `PyBuffer_SizeFromFormat()` should be inlined like this. Please check it for failure before the `PyLong_FromSsize_t()` call.

Reply:
Oh right, NULL should be returned if the result is -1.